modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/info/item/provider/test/ObjectEntryInfoItemFormProviderTest.java

```diff
@@ -179,11 +356,63 @@ private ObjectFieldSetting _createObjectFieldSetting(
 		return objectFieldSetting;
 	}
 
+	private ObjectState _getObjectState(
+		ListTypeEntry listTypeEntry, ObjectStateFlow objectStateFlow) {
+
+		return _objectStateLocalService.fetchObjectStateFlowObjectState(
+			listTypeEntry.getListTypeEntryId(),
+			objectStateFlow.getObjectStateFlowId());
+	}
+
+	private ServiceContext _getServiceContext(ObjectEntry objectEntry)
+		throws Exception {
+
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext(
+				TestPropsValues.getGroupId());
+
+		MockHttpServletRequest mockHttpServletRequest =
+			new MockHttpServletRequest();
+
+		LayoutDisplayPageProvider<?> layoutDisplayPageProvider =
+			_layoutDisplayPageProviderRegistry.
+				getLayoutDisplayPageProviderByClassName(
+					_childObjectDefinition.getClassName());
+
+		mockHttpServletRequest.setAttribute(
+			LayoutDisplayPageWebKeys.LAYOUT_DISPLAY_PAGE_OBJECT_PROVIDER,
+			layoutDisplayPageProvider.getLayoutDisplayPageObjectProvider(
+				new InfoItemReference(
+					_childObjectDefinition.getClassName(),
+					objectEntry.getObjectEntryId())));
+
+		serviceContext.setRequest(mockHttpServletRequest);
+
+		return serviceContext;
+	}
+
 	private ObjectDefinition _childObjectDefinition;
 
 	@Inject
 	private InfoItemServiceRegistry _infoItemServiceRegistry;
 
+	@Inject
+	private LayoutDisplayPageProviderRegistry
+		_layoutDisplayPageProviderRegistry;
+
+	@DeleteAfterTestRun
+	private ListTypeDefinition _listTypeDefinition;
+
+	@Inject
+	private ListTypeDefinitionLocalService _listTypeDefinitionLocalService;
+
+	private ListTypeEntry _listTypeEntry1;
+	private ListTypeEntry _listTypeEntry2;
+	private ListTypeEntry _listTypeEntry3;
```

Comment:
Could you let me know why you didn't delete these variables?

Reply:
Because I forgot

Reply:
I updated the code